ocaml/libs/xapi-stdext/lib/xapi-stdext-unix/unixext.mli

```diff
@@ -146,15 +146,54 @@ val try_read_string : ?limit:int -> Unix.file_descr -> string
 
 exception Timeout
 
-val time_limited_write : Unix.file_descr -> int -> bytes -> float -> unit
+module Timeout : sig
+  type t = private Mtime.Span.t
```

Comment:
Again, introduce the abstraction in some words. It's a time span.

Reply:
module dropped.